### ISSUE TITLE
verificationExceptionForNearMisses and running non localhost failure

### ISSUE THREAD

**Issue by onekilo79** [tomakehurst/wiremock#491]
version 2.1.9
Wiremock -verificationExceptionForNearMisses -  line 387

findNearMissesFor always uses the defaultInstance.

This means that if you have any problem with the setup of your expectations for a remote wiremock instance it will try and look for the localhost wiremock near miss. This gives a misleading error message.

```
WireMock wireMock = new WireMock('http://example.com', 80);
wireMock.verifyThat(1, postRequestedFor(urlMatching('/api/callnotfound')));
````

**Comment by tomakehurst** [comment]
Yeah, I'm aware of this. Will fix in the next release.

**Comment by tomakehurst** [comment]
Fixed by 7c7f71b53d601f5c8258741694d8b14342d584a5
